### PR TITLE
Add usb-filter parameter to .vv

### PR DESCRIPTION
**Pull request by bond95**
@mareklibra Please review https://github.com/oVirt/ovirt-web-ui/issues/273

### CODE REVIEW

File: src/sagas.js [hunk 1]
```diff
@@ -618,13 +622,15 @@ function adjustVVFile ({ data, options }) {
     logDebug('secure-attention was not found, inserting ', text)
     data = data.replace(/^\[virt-viewer\]$/mg, `[virt-viewer]\n${text}`) // ending \n is already there
   }
-
+  if (usbFilter) {
+    data = data.replace(/^\[virt-viewer\]$/mg, `[virt-viewer]\nusb-filter=${usbFilter}\n${text}`)
```
Comment (jniederm) [inline src/sagas.js]: Is it possible that line 623 adds `text` and then `text` is unnecessarily added once again by this `replace()`?

**Comment by mareklibra** [comment]
LGTM except the one issue above

**Comment by bond95** [comment]
Done, @jniederm please review

**Comment by jniederm** [comment]
LGTM

**Pull request merged**